application/frontend/controllers/MfaController.php

```diff
@@ -261,7 +286,7 @@ public function actionDeleteCredential(int $mfaId, int $webauthnId)
      */
     public function actionUpdate(int $id)
     {
-        $mfa = $this->getRequestedMfa($id);
```

Comment:
Was this change intentional? I wasn't sure if it was just the result of an IDE-assisted rename, or if the purpose of the `getRequestedMfa()` function changed, and so a different function needed to be called in order to do what `actionUpdate()` was previously using `getRequestedMfa()` to do.

Reply:
I suppose the name of `getRequestedMfa()` should change (or both of them).  It calls `getAuthorizedMfa()` and then also does some other things.

`actionUpdate` only needed the mfa without the other things loaded onto it.

Maybe we should name it `getAuthorizedMfaWithExtras()`.  If we change the name of `getAuthorizedMfa()`, then of course the name of this function could be made to correspond to that new name.